skbio/stats/power.py

```diff
@@ -821,23 +770,30 @@ def _calculate_power(p_values, alpha=0.05):
 
     """
 
-    w = (p_values < float(alpha)).sum()/float(p_values.shape[0])
+    if len(p_values.shape) == 1:
```

Comment:
It's great to see how effectively code reviews spread knowledge across developers :)